package-res/reportviewer/paginationTemplate.html

```diff
@@ -34,8 +34,8 @@
 <div>
   <div class="page-control">
     <div class="page-control-nav">
-      <span id="left" onclick="left()"><&nbsp;</span>
-      <span id="right" onclick="right()">>&nbsp;</span>
+      <span id="left"><&nbsp;</span>
+      <span id="right">>&nbsp;</span>
```

Comment:
Oh, I see what is going on. Please encode the > and < characters, so that HTML parsers are happy. And literal > or < should be encoded as &lt; and &gt;